euphrates-0.5.0/seeds.txt

```diff
@@ -0,0 +1,4 @@
+fe5709ec8bf754b6f03ce543c543f1af6856ad65@43.199.152.117:26656
```

Comment:
Seems that seeds are the same as peers, which seems weird. Is this the case @filippos47 ? 

If we don't have any seeds that's fine, we can avoid having this file at all. People can sync just with peers.